src/plugins/data/server/search/session/types.ts

```diff
@@ -31,6 +32,7 @@ export interface IScopedSearchSessionsClient<T = unknown> {
   cancel: (sessionId: string) => Promise<{}>;
   delete: (sessionId: string) => Promise<{}>;
   extend: (sessionId: string, expires: Date) => Promise<SavedObjectsUpdateResponse<T>>;
+  getConfig: () => SearchSessionsConfigSchema | null;
```

Comment:
SearchSession still leave in x-pack, but ese strategy needs `defaultExpiration` from its config